README.md

```diff
@@ -1,19 +1,35 @@
+![Title](images/Hauler.svg)
+
+# Hauler
+
 [![Download](https://api.bintray.com/packages/thefuntastyops/hauler/core/images/download.svg) ](https://bintray.com/thefuntastyops/hauler/core/_latestVersion)
 ![minSdk:21](https://img.shields.io/badge/minSDK-21-brightgreen.svg)
 [![Android Arsenal]( https://img.shields.io/badge/Android%20Arsenal-Hauler-brightgreen.svg?style=flat )]( https://android-arsenal.com/details/1/7359 )
 ![Check Master](https://github.com/futuredapp/hauler/workflows/Check%20Master/badge.svg?branch=master)
-![](https://raw.githubusercontent.com/thefuntasty/hauler/master/images/logo_stretch.png)
-
-Hauler
-======
+![License](https://img.shields.io/github/license/futuredapp/hauler?color=black)
 
-Hauler is a library containing custom layout which enables to easily create swipe to dismiss `Activity`.
+Hauler is an Android library containing custom layout which enables to easily create swipe to dismiss `Activity`.
 Implementation is based on code from project [Plaid](https://github.com/nickbutcher/plaid).
 
 ![Alt text](https://github.com/thefuntasty/hauler/blob/master/images/example.gif)
 
-How to use
-----------
+# Installation
+
+```groovy
+dependencies {
+    implementation("com.thefuntasty.hauler:core:latestVersion")
+
+    // optional dependency with set of Data Binding adapters
+    implementation("com.thefuntasty.hauler:core:databinding:latestVersion")
+}
+```
+
+# Features
+
+Hauler library comes with highly customizable `HaulerView` which provides swipe to dismiss functionality. 
+It also ships with `databinding` module which contains Binding Adapters for smoother experience with Android Data Binding implementation.
```

Comment:
ships -> provides?

Reply:
I think it would change the meaning. "Ships" is better in this context as it is a separate artefact in maven repo.